README.md

```diff
@@ -75,22 +86,31 @@ If you want to limit the classes you to be scrubbed you can set the `SCRUB_CLASS
 rake scrub SCRUB_CLASSES=Blog,Post
 ```
 
-If you want to skip the afterhook
+If you want to skip the beforehook
 
 ```
-rake scrub SKIP_AFTERHOOK=true
+rake scrub SKIP_BEFOREHOOK=true
 ```
 
+If you want to skip the afterhook
 
+```
+rake scrub SKIP_AFTERHOOK=true
+```
 
 ### Extending
 
-You may find the need to extend or add additional generators or an after_hook
+You may find the need to extend or add additional generators or an before_hook/after_hook
 
 ```ruby
 ActsAsScrubbable.configure do |c|
   c.add :email_with_prefix, -> { "prefix-#{Faker::Internet.email}" }
 
+  c.before_hook do
+    puts "Running before scrub"
+    raise "Don't run in production" if Rails.env.production?
```

Comment:
will the env ever be production? the other PR assumes it is always development in the `anonymizer_check` task...
is there a way to instead look at something that is not configurable, like database name, or server, or?

Reply:
on further thought, this suggestion makes no sense. it's good as-is.